README.md

```diff
@@ -1,25 +1,35 @@
+[![SublimeHQ Discord](https://img.shields.io/discord/280102180189634562?label=SublimeHQ%20Discord&logo=discord&style=flat-square)](https://forum.sublimetext.com/t/resources-and-bug-tracking/22985)
+
 # Sublime Packages
 
-> **These packages are developed against the latest [Sublime Text 3 Dev Build](http://sublimetext.com/3dev). Bugs may exist on older builds, and the format used is not compatible with builds older than 3092.**
+> **These packages are developed against the latest dev build of Sublime Text, currently build 4094. These syntaxes may be incompatible, sometimes in very subtle ways, with older builds.**
```

Comment:
This is now out of date. Do you want to mention a specific version or say "4xxx series" ?

Reply:
I simply made all changes Will suggested. I assume the `Build 4094` requirement still stands, though.

Reply:
Fair enough. I was going off the line in the 4095 changelog that says

> - Resolve some bugs with syntax definitions that use branching